tests/libwnbd_tests/mock_wnbd_daemon.cc

```diff
@@ -42,6 +42,16 @@ void MockWnbdDaemon::Start()
         WnbdProps.Flags.FlushSupported = 1;
     }
 
+    if (InstanceName.find("TestSetNaaId") != std::string::npos) {
```

Comment:
Let's avoid setting flags based on the instance name. ``SetNaaIdentifier`` is unused, you could either go with that or add another field to the constructor.